datastructures/avl/avl_test.go

```diff
@@ -0,0 +1,172 @@
+package avl
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Can you not do it using the standard library?

It seems the `testify` package has a lot of dependencies. I think its easily testable without this package.

Reply:
removed the ``testify`` dependencies.